tensorflow/workspace2.bzl

```diff
@@ -201,19 +201,24 @@ def _tf_repositories():
     tf_http_archive(
         name = "mkl_dnn_acl_compatible",
         build_file = "//third_party/mkl_dnn:mkldnn_acl.BUILD",
-        patch_file = ["//third_party/mkl_dnn:onednn_acl_threadcap.patch", "//third_party/mkl_dnn:onednn_acl_fixed_format_kernels.patch", "//third_party/mkl_dnn:onednn_acl_depthwise_convolution.patch"],
-        sha256 = "fc2b617ec8dbe907bb10853ea47c46f7acd8817bc4012748623d911aca43afbb",
-        strip_prefix = "oneDNN-2.7",
-        urls = tf_mirror_urls("https://github.com/oneapi-src/oneDNN/archive/v2.7.tar.gz"),
+        patch_file = [
+        "//third_party/mkl_dnn:onednn_acl_threadcap.patch",
+        "//third_party/mkl_dnn:onednn_acl_fixed_format_kernels.patch",
+        "//third_party/mkl_dnn:onednn_acl_depthwise_convolution.patch",
+        "//third_party/mkl_dnn:onednn_acl_threadpoolscheduler.patch"
+        ],
```

Comment:
I think you forgot to add `acl_openmp_fix.patch` back.

Reply:
Sorry, yes. I should have given this a bit more thought. Perhaps it would just make more sense to rebase once https://github.com/tensorflow/tensorflow/pull/59253 is in? (or vice versa)

Reply:
Yes, I think we should wait and rebase.